dkulib/dku_config/dss_parameter.py

```diff
@@ -21,8 +21,9 @@ class DSSParameter:
         required(bool, optional): Whether the value can be None
         cast_to(type, optional): The type to cast the variable in
         cast_to(Any, optional): The default value of the variable (If value is None)
+        label(str, optional): The name displayed to the end user
     """
-    def __init__(self, name: str, value: Any, checks: List[dict] = None, required: bool = False, cast_to: type = None, default: Any = None):
+    def __init__(self, name: str, value: Any, checks: List[dict] = None, required: bool = False, cast_to: type = None, default: Any = None, label: str = None,):
```

Comment:
I like to keep a logical order to the arguments (This is very subjective, you can change it if you don't agree):
```suggestion
    def __init__(self, name: str, value: Any, label: str = None, checks: List[dict] = None, required: bool = False, default: Any = None, cast_to: type = None):
```

If doing so, please also change the order of the documentation

Reply:
done in https://github.com/dataiku/dss-plugin-dkulib/pull/17/commits/ffaa0fc4a91aeb0ffb4d10413ed835bf4b5097ce